### ISSUE TITLE
Broken UI fonts for Libreoffice flatpak and other applications

### ISSUE THREAD

**Issue by nikostourvas** [ublue-os/bluefin#1558]
### Describe the bug

Just updated to Bluefin 39.20240728.0 and Libreoffice UI fonts are broken. Title fonts in secondary windows of firefox (e.g. history window) and thunderbird (e.g. notifications) are also broken. Everything was fine in Bluefin 39.20240721.0.

![image](https://github.com/user-attachments/assets/dac493e4-194e-4bd4-a412-ec1612878300)
![image](https://github.com/user-attachments/assets/37867394-9c83-4b65-b55e-042e430b96b5)

### What did you expect to happen?

All UI elements in application to look properly as they did in the previous bluefin version (39.20240721.0) that I had been using for the past week.

### Output of `rpm-ostree status`

```shell
State: idle
AutomaticUpdates: stage; rpm-ostreed-automatic.timer: no runs since boot
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:39
                   Digest: sha256:7291ba37269477ecac9e2e6902fabb71a3dbb8fc8b434c2aae26df9dc15b5f9a
                  Version: 39.20240728.0 (2024-07-28T05:53:12Z)
          LayeredPackages: cairo-gobject-devel gobject-introspection-devel grubby
                           python3-cairo-devel python3-pytest
            LocalPackages: nautilus-dropbox-2023.09.06-1.fc37.x86_64

  ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:39
                   Digest: sha256:f8a12908108c8c77a2319171c3c2823dd863db51cfba4a4f6460cbf20bafa473
                  Version: 39.20240721.0 (2024-07-21T05:53:17Z)
          LayeredPackages: cairo-gobject-devel gobject-introspection-devel grubby
                           python3-cairo-devel python3-pytest
            LocalPackages: nautilus-dropbox-2023.09.06-1.fc37.x86_64

  ostree-image-signed:docker://ghcr.io/ublue-os/bluefin-dx:39
                   Digest: sha256:fa59a72899d05f7bdd5919610f20dac1838a489e0801227efb24f499a381f890
                  Version: 39.20240312.0 (2024-03-12T16:53:42Z)
          LayeredPackages: cairo-gobject-devel gobject-introspection-devel grubby
                           python3-cairo-devel python3-pytest
            LocalPackages: nautilus-dropbox-2023.09.06-1.fc37.x86_64
                   Pinned: yes
```

### Output of `groups`

```shell
nikos wheel lxd libvirt incus-admin docker
```

**Comment by castrojo** [comment]
I haven't been able to reproduce this but here's some workarounds people are working with: https://universal-blue.discourse.group/t/encoding-issue-in-bluefin/3281/18

**Comment by castrojo** [comment]
This one seems to be fixing it for folks:

    flatpak list --columns=application | xargs -I %s -- flatpak run --command=fc-cache %s -f -v

**Comment by castrojo** [comment]
OK this should be live for gts users, unfortunately there's some nvidia and -dx skew so I'll need to try to rebuild them again tomorrow. 

The fix runs the fc-cache command on the next login in the background.

**Comment by castrojo** [comment]
Anyone still getting this?

**Comment by 0xBLERKO** [comment]
I am. Tried the suggested command to run fc-cache for apps and the system, no change. I used gnome tweak to change from ubuntu font to default gnome cantarell and that resolved for me. Changing back to ubuntu font recreates the issue. It appears user specific as other user profiles on the same system do not have the font issue and display ubuntu font just fine. Some kind of user font cache not cleared by fc-cache?
